WMIAdventure/backend/WMIAdventure_backend/IngameUsers/urls.py

```diff
@@ -1,12 +1,14 @@
 from django.urls import path
 
-from .views import UserDecksView, PaginatedUsersView, UserView, UserDeckView, UserLevelView, UserCardsView
+from .views import UserDecksView, PaginatedUsersView, UserView, UserDeckView, UserLevelView, UserCardsView, \
+    UpgradeCardView
 
 urlpatterns = [
     path('', PaginatedUsersView.as_view(), name='user profiles'),
     path('<int:pk>/', UserView.as_view(), name='user profile'),
     path('<int:pk>/level/', UserLevelView.as_view(), name='user level info'),
     path('<int:pk>/cards/', UserCardsView.as_view(), name='user-cards'),
+    path('<int:pk>/cards/upgrade/<int:card_id>/', UpgradeCardView.as_view(), name='upgrade-card'),
```

Comment:
dziwnie się tego używa, jak mamy np endpoint to akceptowania karty to jest `<id>/accept/` a tu jest odwrotnie, może zmienimy?